src/RelayClient.ts

```diff
@@ -125,6 +126,15 @@ class RelayClient extends EnvelopingEventEmitter {
 
     return BigNumber.from(gasPrice.toFixed());
   };
+  public async isSmartWalletOwner(smartWalletAddress: string, owner:string): Promise<boolean>{
+    const iForwarder = IForwarder__factory.connect(smartWalletAddress, this._provider);
+    if(await iForwarder.getOwner()  !== utils.solidityKeccak256(['address'], [owner])){
+      return false;
+    }
+
+    return true;
```

Comment:
Over here Im not sure if I should return a flag or throw an error. @rsksmart/payments

Reply:
The goal of the method is to check if it is the owner or not. I think your current approach is correct. If an error is required to be thrown, the caller  should throw it.

Reply:
👆🏽